examples/todomvc/src/components/TimeTravel.tsx

```
@@ -1,8 +1,8 @@
 const shortLink = (l: string) => `${String(l).slice(0, 4)}..${String(l).slice(-4)}`;
 const clockLog = new Set<string>();
 
-export const TimeTravel = ({ ledger }) => {
-  ledger.clock && ledger.clock.length && clockLog.add(ledger.clock.toString());
+export const TimeTravel = ({ database }) => {
+  database.clock && database.clock.length && clockLog.add(database.clock.toString());
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)